app/models/package_manager/go.rb

```diff
@@ -21,6 +21,8 @@ class Go < Base
       ".svn",
     ].freeze
 
+    VERSION_MODULE_REGEX = /(.+)\/(v\d)/.freeze
```

Comment:
I suppose the day could come when people exceed single digits 😱 

```suggestion
    VERSION_MODULE_REGEX = /(.+)\/(v\d+)/.freeze
```

Reply:
Oh totally great point! I think we're close already.